### DISCUSSION TITLE
Buttons for web client in toolbar

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#5023] [Ideas]
It would be great if we had buttons to launch the web client from the toolbar of the gui clients.

**Comment by ckerr** [comment]
There's a button to launch it from the GTK and Qt clients' preferences settings. I _think_ there's one in the macOS client as well?

I'd vote against putting it in the toolbar; that's prime, finite real estate. That should be reserved for things like "start" and "stop", not "launch web client" :smile_cat:

**Reply by GaryElshaw** [reply]
Yes, you have to click through the preferences to get there, i thought something up-front might be good.
